styles.css

```diff
@@ -78,7 +78,312 @@
+    margin: 24px 0;
+  }
+
+  hr.new1 {
+    width: 15%;
+    height: 4px;
+    left: 139.5px;
+    top: 121px;
+    background-color: #36b37e;
+    border-radius: 24px;
+  }
+
+  .imgholder {
+    background-color: #ebf0ee;
+    width: 100%;
+    height: 222px;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .card-one {
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .articlewraper {
+    display: grid;
+    row-gap: 24px;
+    padding: 10px 10px;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+  }
+
+  .project-languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    list-style: none;
+    gap: 5px;
+  }
+
+  .lang-rr {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    font-family: 'Inter', Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #3a4a42;
+  }
+
+  .lang-cs {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    font-family: 'Inter', Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #3a4a42;
+  }
+
+  .lang-js {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    font-family: 'Inter', Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #3a4a42;
+  }
+
+  .lang-ht {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    font-family: 'Inter', Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #3a4a42;
+  }
+
+  .btn-card {
+    border: 0 solid transparent;
+    background: #36b37f;
+    text-decoration: none;
+    font-size: 17px;
+    color: #fff;
+    letter-spacing: 0.03em;
+    padding: 12px;
+    border-radius: 4px;
+    font-weight: 500;
+    margin-top: 13px;
+    cursor: pointer;
+  }
+
+  .about_section {
+    background-image: url('Images/illustration1-about-me-mobile.svg'), url('Images/illustration2-about-me-mobile.svg');
+    background-position: top right, bottom left;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 0;
+    width: 351px;
+    height: 333px;
```

Comment:
The about me section takes fixed width which is not recommended as we want to implement a responsive design. 

- [ ] Please consider to not use fix width for big items and sections in our design;

Your design: 
![image](https://user-images.githubusercontent.com/83596135/159105086-919da4bc-78d2-4e53-8fed-d1f30d33c680.png)